### DISCUSSION TITLE
Day of Dragon

### DISCUSSION THREAD

**Discussion by mvaessen** [gameserverapp/Platform#28] [Game request]
https://steamcommunity.com/app/1088090/discussions/1/4737210531942775567/

**Comment by Decollates** [comment]
The game has now left alpha and its full 1.0 release is on Steam. 
https://store.steampowered.com/app/1088090/Day_of_Dragons/

From their discord, here https://discord.gg/kPYAmzUQ 
![image](https://github.com/gameserverapp/Platform/assets/44717221/b46291c2-55d4-457b-a0c8-cc438288994b)

**Comment by ssiwm** [comment]
That would be great, I've been following this game from the beginning.
